### PR TITLE
Graph Eq: Batch Norm support

### PR DESCRIPTION
**Pull request by Giuseppe5**
This depends on #530

### CODE REVIEW

File: src/brevitas/graph/equalize.py [hunk 1]
```diff
@@ -159,6 +165,8 @@ def _cross_layer_equalization(srcs, sinks):
         src_broadcast_size = [1] * module.weight.ndim
         src_broadcast_size[axis] = module.weight.size(axis)
         module.weight.data = module.weight.data * torch.reshape(scaling_factors, src_broadcast_size)
+        if isinstance(module, (torch.nn.BatchNorm1d, torch.nn.BatchNorm2d, torch.nn.BatchNorm3d)):
+            module.running_mean.data = module.running_mean.data * torch.reshape(inverse_scaling_factors, src_broadcast_size)
```
Comment (volcacius) [inline src/brevitas/graph/equalize.py:169]: We should modify only the learned parameters to avoid interfering with further statistics collection

File: tests/brevitas/graph/equalization_fixtures.py [hunk 2]
```diff
@@ -4,6 +4,40 @@
 import torch.nn as nn
 
 
+@pytest_cases.fixture
+def bnconv_model():
+    class BNConvModel(nn.Module):
+        def __init__(self) -> None:
+            super().__init__()
+            self.bn = nn.BatchNorm2d(3)
```
Comment (volcacius) [inline tests/brevitas/graph/equalization_fixtures.py:12]: The learned parameters should also have non trivial init

File: tests/brevitas/graph/equalization_fixtures.py [hunk 3]
```diff
@@ -4,6 +4,40 @@
...
+
+
+@pytest_cases.fixture
+def convbn_model():
+    class ConvBNModel(nn.Module):
+        def __init__(self) -> None:
+            super().__init__()
+            self.conv = nn.Conv2d(3, 128, kernel_size=3)
+            self.bn = nn.BatchNorm2d(128)
+            # Simulate statistics gathering
```
Comment (volcacius) [inline tests/brevitas/graph/equalization_fixtures.py:30]: Same as above

**Review by volcacius — CHANGES_REQUESTED** [review]

**Review by volcacius — APPROVED** [review]
